streams/ro.m3u

```diff
@@ -128,6 +136,8 @@ https://5ab29cc78f681.streamlock.net/NasulTV/livestream/playlist.m3u8
 http://novapress.ro:2022/live/nova/playlist.m3u8
 #EXTINF:-1 tvg-id="ObservatorNews.ro",Observator News (720p)
 https://live1ag.antenaplay.ro/live_ObservatorNews/live_ObservatorNews.m3u8
+#EXTINF:-1 tvg-id="PartyMix.ro",Party Mix (1080p)
+http://103.145.13.15:25461/live/promo/promo/2.m3u8
```

Comment:
Not quite sure. It's an Xtream-Codes but seems to be owned by the company that has this channel.